bin/sign

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env ruby
+
+ENV["PROJECT_ROOT"] = Dir.pwd
```

Comment:
What is this being used for?

Reply:
That's needed for packaging. It's set in the other binaries too. (Packaging fails with no project root set)

Reply:
ahh gotcha, good to know!